prov/util/src/util_mem_monitor.c

```diff
@@ -153,6 +145,18 @@ void ofi_monitors_init(void)
 	if (!cache_params.max_size)
 		cache_params.max_size = ofi_default_cache_size();
 
+	/* At this time, the import monitor could have set the default monitor,
+	 * do not override by default */
```

Comment:
Please use the same comment style as shown in the copyright notice at the beginning of the fie. The phrase "by default" can be removed.

Reply:
Done